ibm/service/power/resource_ibm_pi_shared_processor_pool.go

```diff
@@ -60,6 +60,11 @@ func ResourceIBMPISharedProcessorPool() *schema.Resource {
 				Required:    true,
 				Description: "PI cloud instance ID",
 			},
+			Arg_HostID: {
+				Description: "The host id of a host in a host group (only available for dedicated hosts)",
```

Comment:
Is this required it will be breaking change 
From docs I see this as optional can we change to optional and handle it
https://cloud.ibm.com/apidocs/power-cloud#pcloud-sharedprocessorpools-post

Reply:
You are correct it is an optional field, I have updated it.